src/main.ts

```diff
@@ -1,11 +1,11 @@
-import { type App, Modal, Plugin, PluginSettingTab, Setting, TFile } from 'obsidian';
```

Comment:
This seems like a formatting issue, `biome check src/ --write` to fix all formatting.